openapi/openapi.yaml

```
@@ -62,4 +64,6 @@ paths:
   /users:
     $ref: paths/users.yaml
   /users/{username}:
-    $ref: paths/users@{username}.yaml
\ No newline at end of file
+    $ref: paths/users@{username}.yaml
+  /healthcheck:
+    $ref: paths/healthcheck.yaml
```

To be consistent with other projects, e.g. https://github.com/nlpsandbox/nlpsandbox-schemas/blob/develop/openapi/commons/paths/healthCheck.yaml

```suggestion
    $ref: paths/healthCheck.yaml
```